### PR TITLE
[PIE-3736] Remove outline on mouse focus

### PR DESCRIPTION
**Pull request by kat3samsin**
## Description

A few sentences describing the overall goals of the pull request. Any special considerations and decisions.

Also include any references to relevant discussions and documentation.

## Checklist

- [ ] This PR has good automated test coverage
- [ ] The storybook for the component has been updated

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Pull down PR.
1. `npm run dev`.
1. Open Storybook.
1. Eat cookies.
1. Verify cookies are delicious.

### CODE REVIEW

File: src/system/Button/Button.js [hunk 1]
```diff
@@ -34,8 +34,7 @@ const Button = React.forwardRef( ( { disabled, onClick, sx, ...props }, forwardR
 				'&:hover': {
 					textDecoration: 'none',
 				},
-				'&:focus': theme => theme.outline,
-				'&:focus-visible': theme => theme.outline,
+				'&focus-visible': theme => theme.outline,
```
Comment (djalmaaraujo) [inline src/system/Button/Button.js]: Did you miss the `:`?

Comment (kat3samsin) [reply src/system/Button/Button.js]: Yeah you're right

Comment (kat3samsin) [reply src/system/Button/Button.js]: Added in 3dbe67165889b810251386f47f59d48349b9950d

**Review by djalmaaraujo — APPROVED** [review]
Thank you @kat3samsin

**Pull request merged**